Dockerfile

```diff
@@ -134,14 +136,17 @@ ENV LD_LIBRARY_PATH $LD_LIBRARY_PATH:/usr/local/cuda/extras/CUPTI/lib64:/usr/loc
 # << END Configure Bazel
 
 
-
-
 # Copy DeepSpeech repo contents to container's /DeepSpeech
 COPY . /DeepSpeech/
 
+# Alternative clone from GitHub 
+# RUN apt-get update && apt-get install -y git-lfs 
```

Comment:
Maybe you could leverage Docker's `ARGS` capability here to expose that more easily? I'm not too knowledgeable of docker, but that would look like a better way than having to comment / uncomment ?

Reply:
Nice idea. Sadly I am afraid that Docker **ARGS** capability not sufficient for that use case. It is possible to make the RUN command conditional, however, the **COPY** step above would still remain. Actually, we would require something like 


```
if 
# Copy DeepSpeech repo contents to container's /DeepSpeech
COPY . /DeepSpeech/
else 
# Alternative clone from GitHub 
# RUN apt-get update && apt-get install -y git-lfs 
```

which - as far as I know - is not possible within a dockerfile. 

What we could do with **ARGS**  is something like: 

```
ARG checkout
RUN if [-n "$checkout"] apt-get update && apt-get install -y git-lfs
```
 
But this would only solve one part of the problem. Taking this into account I think the uncomment solution is kind of poorman but still the most practical one. Moving this into another dockerfile would make sure that not all changes end up in both dockerfiles. Indeed it might be a rare use case to clone the DeepSpeech repo from GitHub but still having it in the dockerfile doesn't hurt anyone.

Reply:
Yeah, I don't think it's a very important problem to solve with complicated tools. Commenting is enough if someone ever would uncomment it)

Reply:
@GeorgeFedoseev so no further changes required?

Reply:
@cfreemoser Can't build myself right now, but yeah seems good to me.